src/resources/index.js

```diff
@@ -135,9 +138,24 @@ module.exports = {
   },
   messages: {
     cli: {
+      installCompiler: (compiler, fromLocal) =>
```

Comment:
can we keep those alphabetic. Perhaps first all lower case (new format) then all upper case (old format to be converted to new)